pkg/clockcache/cache.go

```diff
@@ -30,9 +32,10 @@ type element struct {
 
 	// CLOCK marker if this is recently used
 	used uint32
+	size uint64
 
 	// pad Elements out to be cache aligned
-	_ [24]byte
+	_ [14]byte
```

Comment:
If I did my may right, you added either 8 or 12 bytes (8 for the `size`, and possibly 4 for padding), but only removed 10 from the padding 🤔. Incidentally, I'd recommend putting `size` before `used` to make this calculation easier.